libraries/bezier-rs/src/bezier/transform.rs

```diff
@@ -49,11 +50,12 @@ impl Bezier {
 		}
 	}
 
-	/// Returns the Bezier curve representing the sub-curve starting at the point corresponding to `t1` and ending at the point corresponding to `t2`.
-	pub fn trim(&self, t1: f64, t2: f64) -> Bezier {
+	/// Returns the Bezier curve representing the sub-curve starting at the point `c1` and ending at the point `c2` along the curve.
```

Comment:
Please specify in this doc comment if it's okay to give c1 and c2 in any order, or if one has to be bigger than the other.